pandas/tests/arrays/test_integer.py

```diff
@@ -710,6 +710,21 @@ def test_to_integer_array_float():
     assert result.dtype == Int64Dtype()
 
 
+@pytest.mark.parametrize(
+    "values, dtype",
+    [
+        ([9999999999999999, 123123123123123123, 10000000000000543, np.nan], "Int64"),
+        (np.array([9999999999999999, 123123123123123123, 10000000000000543],
+         dtype=np.int64), "Int64"),
+    ],
+)
+def test_to_integer_array_large_numbers(values, dtype):
+    result = integer_array(values, dtype=dtype)
+    expected = integer_array(values, dtype="Int64")
+    assert result.dtype == dtype
+    tm.assert_extension_array_equal(result, expected, check_exact=True)
```

Comment:
I _think_ should probably replicate issue with Series constructor, even though the issue is with the array constructor and use `assert_series_equal`

Reply:
Okay, then should I add that tests here or somewhere else?

Reply:
I think in this module is fine. There are tests near the bottom that use Series and DataFrames. could move to the bottom.